client/dashboard/sites/index.tsx

```diff
@@ -3,13 +3,17 @@ import {
 	userPreferenceQuery,
 	userPreferenceMutation,
 	sitesQuery,
+	siteBySlugQuery,
+	siteByIdQuery,
+	queryClient,
```

Comment:
Should we import like this, or should we use `useQueryClient()` 🤔 (not sure)

Reply:
Updated!